src/mlpack/tests/gan_test.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include <mlpack/methods/ann/init_rules/gaussian_init.hpp>
 #include <mlpack/methods/ann/loss_functions/cross_entropy_error.hpp>
+#include <mlpack/methods/ann/loss_functions/sigmoid_cross_entropy_error.hpp>
 #include <mlpack/methods/ann/gan/gan.hpp>
```

Comment:
It looks like you have removed use of `cross entropy error` from the file. So you can remove the `cross_entropy_error.hpp` header from file. 
`Note`:This is applicable if it is correct to have all `FFNs` `sigmoid cross entropy error` which will be decided by zoq and rcurtin.